Scripts/Modules/Highlight/HighlightModule.cs

```diff
@@ -24,7 +24,7 @@ void LateUpdate()
 				foreach (var m in go.GetComponentsInChildren<MeshFilter>())
 				{
 					var highlightTransform = m.transform;
-					Matrix4x4 highlightScaleIncreaseMatrix = Matrix4x4.TRS(highlightTransform.position, highlightTransform.rotation, highlightTransform.lossyScale + kHighlightScaleIncrease);
+					Matrix4x4 highlightScaleIncreaseMatrix = Matrix4x4.TRS(highlightTransform.position, highlightTransform.rotation, highlightTransform.lossyScale + k_HighlightScaleIncrease);
```

Comment:
I just realized this also wants a * Viewer.GetViewerScale(). We can add this later.